collections/made-in-africa/index.md

```diff
@@ -19,6 +19,7 @@ items:
  - beopencloud/cno
  - javascriptdata/danfojs
  - bangajs/banga-cli
+ - jembi/openhim-core-js/
```

Comment:
```suggestion
 - jembi/openhim-core-js
```
This shouldn't have a trailing `/` on it. Please update w/this suggestion and I'll merge!